ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/util/BlobSidecarPoolImpl.java

```diff
@@ -302,6 +279,44 @@ public synchronized int getTotalBlobSidecarsTrackers() {
     return blockBlobSidecarsTrackers.size();
   }
 
+  private BlockBlobSidecarsTracker internalOnNewBlock(final SignedBeaconBlock block) {
+    final SlotAndBlockRoot slotAndBlockRoot = block.getSlotAndBlockRoot();
+
+    final BlockBlobSidecarsTracker tracker =
+        getOrCreateBlobSidecarsTracker(
+            slotAndBlockRoot,
+            newTracker -> {
+              newTracker.setBlock(block);
+              onFirstSeen(slotAndBlockRoot);
+            },
+            existingTracker -> {
+              if (!existingTracker.setBlock(block)) {
+                // block was already set
+                return;
+              }
+
+              if (existingTracker.isFetchTriggered()) {
+                // block has been set for the first time and we fetched missing blobSidecars, so we
```

Comment:
`and we fetched missing blobSidecars` maybe something like `and we are trying to fetch missing blobSidecars` or `and we are requested fetch of missing blobSidecar`? I'm not native english-speaking but current comment sounds like the done thing.

Reply:
yeah, `fetched` isn't particularly accurate here. fixing it